c_src/message.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <limits.h>
+
+struct Message {
```

Comment:
I would just incorporate this struct into the `notif_set` header as it currently exists mostly for interaction with the `notif_set` structure.